source/common/filesystem/inotify/watcher_impl.cc

```diff
@@ -11,6 +9,7 @@
 
 #include "common/common/assert.h"
 #include "common/common/utility.h"
+#include "common/filesystem/watcher_impl.h"
```

Comment:
Sorry I missed this on the last pass but I think this include change still needs to be reverted